precompiles/parachain-staking/src/lib.rs

```diff
@@ -249,6 +253,7 @@ impl<Runtime> ParachainStakingWrapper<Runtime>
 where
 	Runtime: parachain_staking::Config + pallet_evm::Config,
 	BalanceOf<Runtime>: EvmData,
+	Runtime::AccountId: Into<Address>,
```

Comment:
@nanocryk is there a way to convert Vec<AccountId> to Vec<Address> without adding this trait bound? Right now the trait bound is causing a runtime error s.t. Precompile is not implemented ParachainStakingWrapper